.github/actions/oc-keycloak-login/action.yml

```diff
@@ -15,3 +15,8 @@ runs:
       working-directory: provision/aws
       env:
         CLUSTER_NAME: ${{ inputs.clusterName }}
+
+    - id: enable-self-describe
+      shell: bash
+      working-directory: provision/aws
+      run: echo "SUT_DESCRIBE=$(pwd)/rosa_describe.sh ${{ inputs.clusterName }}" >> "$GITHUB_ENV"
```

Comment:
is there a potential to extend this for non-ROSA OCP or K8S clusters ?

Reply:
Yes, as long as it returns any JSON. The contents and schema of the `system_under_test` node will depend on the system being described and the script being used.

Reply:
thanks, it would be a great next issue.